tests/system/action/motion/test_update.py

```diff
@@ -463,6 +463,7 @@ def test_update_permission(self) -> None:
                 "title": "title_bDFsWtKL",
                 "text": "text_eNPkDVuq",
                 "reason": "reason_ukWqADfE",
+                "created": 1686735327,
```

Comment:
As being meta data check whether you could change the `created` field with `motion.can_manage_metadata` in your white list tests.

Reply:
hehehe, it is metadata, because you changed it 1 hour ago. Anyway, I have added a test and the permission.